bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -37,82 +51,48 @@
-            <div class="screens-sprite"></div>
-          </div>
-
-          <ul>
-            <li><a href="#" id="scroll-homescreen" class="active-state">{{_('Scroll the homepage')}}</a></li>
-            <li><a href="#" id="swipe-apps">{{_('Swipe between apps')}}</a></li>
-          </ul>
-        </div> #}
       </div>
     </section>
     <section class="fxos-features">
       <div class="content-wrapper">
         <h2>{{_('Smart has never been so simple')}}</h2>
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/scroll_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Scroll for what you need')}}</h3>
-          <p>{{_('Say goodbye to multiple home screens. Everything you need is right there in one clean, continuous scroll. What could be easier?')}}</p>
-        </div>
-
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/search_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Search in a snap')}}</h3>
-          <p>{{_('Search the Web and the Firefox Marketplace, or visit a website directly —  all from the search bar at the top of your home screen.')}}</p>
-        </div>
-
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/swipe_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Swipe&nbsp;between apps')}}</h3>
-          <p>{{_('Access your recently used apps in a flash. Just swipe from the edge of the screen to cycle through all of your open apps.')}}</p>
-        </div>
+        <ul class="features">
+          <li>
+            <div class="icon scroll"></div>
+            <h3>{{_('Scroll for what you need')}}</h3>
+            <p>{{_('Say goodbye to multiple home screens. Everything you need is right there in one clean, continuous scroll. What could be easier?')}}</p>
+          </li>
+          <li>
+            <div class="icon search"></div>
+            <h3>{{_('Search in a snap')}}</h3>
+            <p>{{_('Search the Web and the Firefox Marketplace, or visit a website directly —  all from the search bar at the top of your home screen.')}}</p>
+          </li>
+          <li>
+            <div class="icon swipe"></div>
+            <h3>{{_('Swipe&nbsp;between apps')}}</h3>
+            <p>{{_('Access your recently used apps in a flash. Just swipe from the edge of the screen to cycle through all of your open apps.')}}</p>
+          </li>
+        </ul>
+        {% if l10n_has_tag('new_fxos_title') %}
+          <p class="hidden">{{_('Firefox OS is designed to fit your life and with Adaptive App Search it does just that. Just start typing to get fast and easy access to related apps and local content. It’s smart and simple — the way every search should be.')}}</p>
```

Comment:
I'm not sure why this is here? Is it ever displayed? Is this just temporary for l10n?